packages/inscription-view/src/test-utils/test-utils.tsx

```diff
@@ -183,7 +183,7 @@ const ContextHelper = (
       }
     }
   };
-  const editorContext = DEFAULT_EDITOR_CONTEXT;
+  const editorContext = structuredClone(DEFAULT_EDITOR_CONTEXT);
```

Comment:
Looks good 👍 I have also seen this sturcturedClone in other places but I don't quite understand what this is good for✌️